cts/CTS.py

```diff
@@ -373,6 +373,31 @@ def install_helper(self, filename, destdir=None, nodes=None, sourcedir=None):
             self.rsh.cp(file_with_path, "root@%s:%s/%s" % (node, destdir, filename))
         return file_with_path
 
+    def uninstall_helper(self, filename, dir=None, nodes=None):
+        if not nodes:
+            nodes = self.Env["nodes"]
+
+        if not dir:
+            dir = CTSvars.CTS_home
+
+        self.debug("Uninstalling %s from %s on %s" % (filename, dir, repr(self.Env["nodes"])))
+        for node in nodes:
+            rc = self.rsh(node, "rm -rf %s/%s" % (dir, filename))
+            if rc != 0:
+                return rc
+        return 0
+
+    def systemctl_daemon_reload(self, nodes=None):
+        if not nodes:
+            nodes = self.Env["nodes"]
+
+        self.debug("Reloading the system manager configuration")
+        for node in nodes:
+            rc = self.rsh(node, "systemctl daemon-reload")
+            if rc != 0:
+                return rc
```

Comment:
I'd move the debug inside the loop, and add the node name, so any error messages can be tied to a particular node. Also, I wouldn't return immediately on failure, b/c the other nodes may succeed. Similarly for the uninstall loop.

Reply:
Done.